.github/workflows/astarte-end-to-end-test-workflow.yaml

```
@@ -63,11 +63,13 @@ jobs:
         wget https://github.com/astarte-platform/astartectl/releases/download/v22.11.02/astartectl_22.11.02_linux_x86_64.tar.gz
         tar xf astartectl_22.11.02_linux_x86_64.tar.gz
         chmod +x astartectl
-    - name: Create realm
+    - name: Generate realm keypair and JWT
       run: |
         ./astartectl utils gen-keypair test
-        ./astartectl housekeeping realms create test --astarte-url http://api.astarte.localhost --realm-public-key test_public.pem -k compose/astarte-keys/housekeeping_private.pem -y
-        echo "E2E_REALM=test" >> $GITHUB_ENV
+        E2E_HOUSEKEEPING_API_JWT=$(./astartectl utils gen-jwt housekeeping -k compose/astarte-keys/housekeeping_private.pem)
+        echo "E2E_HOUSEKEEPING_API_JWT=$E2E_HOUSEKEEPING_API_JWT" >> $GITHUB_ENV
+
+        echo "E2E_REALM_PUBLIC_KEY_PEM=$(pwd)/test_public.pem" >> $GITHUB_ENV
```

this won't work, because the e2e test is run inside docker and has no access to external files (unless you monut them)
I suggest you just use the file contents and call it a day

```suggestion
        echo "E2E_REALM_PUBLIC_KEY_PEM=$(cat ./test_public.pem)" >> $GITHUB_ENV
```